Echo swiftlint version

nacho4d opened the pull request:
## Description
Echo swiftlint version before swiftlint-ing files

## Motivation and Context

Swiftlint is great but it *adds/changes/removes/deprecates rules* changing the behaviour quite frequently. 

If the developer wants to reproduce same warnings/errors in local computer having a version in the logs helps a lot. This little PR should write something like this:

> Running linter swiftlint version 0.24.0

## How Has This Been Tested?

I just partially executed the script in my local machine

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have submitted a [CLA form](https://github.com/IBM-Swift/CLA) ->
- [ ] If applicable, I have updated the documentation accordingly.
- [ ] If applicable, I have added tests to cover my changes.

nacho4d committed:
echo swiftlint version

rolivieri commented:
@nacho4d  Thanks for the PR! Before we can process the PR, can you go over the checklist below? We now have a PR template (when you opened the PR we did not have one in place). If you have contributed before our GitHub org, then you should have a CLA in place (if my mind is not failing me, you may have one in place since you contributed to Kitura recently):

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have submitted a [CLA form](https://github.com/IBM-Swift/CLA)

nacho4d commented:
@rolivieri I have updated the description of the PR to align with the new template.
Yes, I have previously submitted PRs to Kitura. And now I added the check mark here too.

(Actually I also work for IBM (in Japan) and by chatting with some members in swift-at-ibm.slack.com they suggested I don't need one a CLA so I actually didn't submit it. 
If needed, my email is: ignacio AT jp DOT ibm DOT com )
https://swift-at-ibm.slack.com/archives/C29BEUQH3/p1508371282000221
